docs/StardustDocs/topics/adjustSchema.md

```diff
@@ -1,10 +1,10 @@
 [//]: # (title: Adjust schema)
 
-`DataFrame` interface has type argument `T` that doesn't affect contents of `DataFrame`, but marks `DataFrame` with a type that represents data schema that this `DataFrame` is supposed to have.
+[`DataFrame`](DataFrame.md) interface has type argument `T` that doesn't affect contents of `DataFrame`, but marks `DataFrame` with a type that represents data schema that this `DataFrame` is supposed to have.
```

Comment:
For consistency, maybe link all these up?

Reply:
or if you're talking about the concept change it into "data frame".
We have either "a DataFrame" (the type, so link it up) instance, "(Kotlin) DataFrame", the library, or "a data frame", the concept. We should be careful when using these.

Reply:
Agree that we need to differ all of them, probably in some places (in the second round of fixes for another missed links)